app/assets/javascripts/oregon_digital/apa_citation_copy.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function () {
+    // Init clipboard-rails button for all copy URL buttons
+    var clipboard = new Clipboard('#apa-copy-button');
```

Comment:
We could use a more generic query, like `.citation-copy-button` to catch all these buttons at once